cmd/tidb-lightning/main.go

```diff
@@ -96,3 +105,47 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func checkSystemRequirement(cfg *config.GlobalConfig) error {
```

Comment:
(1) This should be checked using the task config before every task starts
(2) This should be ignorable using `--check-requirements=false`